### ISSUE TITLE
Overflow issues

### ISSUE THREAD

**Issue by mechite** [pterodactyl/panel#2783]
**Background (please complete the following information):**
Not required, applies to all 1.x.x versions

**Describe the bug**
This one is not really a bug, but doesn't fall under the ideas category.

When you go to the main screen of the panel, there is no scrollbar, however most pages actually have a scrollbar, leading to page switching giving you this weird push where the scrollbar is created. This is bad UX and an easy fix would be to just enable the scrollbar on the main page, as it doesn't do much apart from push everything so it matches the rest of the panel.

**To Reproduce**
1. Go to the main page of the panel.
2. Attempt to open a servers' page.
3. You'll see a newly created overflow bar / scrollbar.

**Expected behavior**
A scrollbar that remains on the page, "sticky", so that it doesn't interfere with UX.

**Comment by mechite** [comment]
Any updates on this issue? I've resorted to removing the scrollbar on my personal panel using some a CSS selector to hide the scrollbar visibility outright however it would be nicer having it usable without my page "bouncing" when I switch.

Also to note that I removed the copyright footer as I would personally like if there was a defined footer like on 0.7.x, sticky to the bottom of the page so that it doesn't impede with the content by floating. If a bar styled element at the bottom similar to the header, with panel version, load speed and copyright was added it would be a lot nicer.
